packages/microsoft-reactnative-sampleapps/windows/SampleAppCPP/SampleAppCpp.vcxproj

```diff
@@ -177,13 +172,13 @@
       <DeploymentContent>false</DeploymentContent>
     </Text>
   </ItemGroup>
+  <PropertyGroup Label="ReactNativeWindowsNodeProps">
+    <ReactNativeWindowsDir Condition="'$(ReactNativeWindowsDir)' == ''">$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'lerna.json'))vnext\</ReactNativeWindowsDir>
+  </PropertyGroup>
   <ItemGroup>
-    <Natvis Include="$(ReactNativeWindowsDir)\Folly\Folly.natvis" />
+    <Natvis Include="$(ReactNativeWindowsDir)Folly\Folly.natvis" />
```

Comment:
Not related to this change, but we dont need this natvis in the samples project.  Folly isn't exposed in the public interface.

Reply:
removed...

---
In reply to: [428692093](https://github.com/microsoft/react-native-windows/pull/4961#discussion_r428692093) [](ancestors = 428692093)